Training24.sln

```diff
@@ -1,5 +1,4 @@
-﻿

```

Comment:
This change should not be here. Please start using git di before adding and committing